### ISSUE TITLE
Remove_beginning1 tool broken (perl: not found) 

### ISSUE THREAD

**Issue by gallardoalba** [galaxyproject/galaxy#12603]
Describe the bug
The Remove_beginning1 tool doesn't work.

Galaxy Version and/or server at which you observed the bug
Galaxy Version: 21.09 (usegalaxy.org). It works fine on 21.05 (usegalaxy.eu)

To Reproduce
History which contains an example: https://usegalaxy.org/u/gallardoalba/h/errorremovebeginnig

Expected behavior
Remove the first line on a file.

Actual behavior
It triggers an error (perl: not found)

Screenshots
![Screenshot from 2021-09-30 15-26-02](https://user-images.githubusercontent.com/26678117/135463748-b29be8f5-7cf6-430f-906b-250a9f57579a.png)

Additional context
Discovered during 21.09 release testing.

**Comment by mvdbeek** [comment]
Duplicate of #12602

**Comment by mvdbeek** [comment]
Actually, we might want to add a requirement to the tool.

**Comment by gallardoalba** [comment]
> Actually, we might want to add a requirement to the tool.

This is also related to the training tag.

**Comment by mvdbeek** [comment]
Yes, which means the tool runs in a container if available. But this tool doesn't specify a dependency, so the default container is used, which only has python available, but not perl.
